### PR TITLE
attribute: document lowercased keys in headers

### PR DESCRIPTION
**Pull request by kyessenov**
Ref: https://github.com/istio/istio/issues/9432
Signed-off-by: Kuat Yessenov <kuat@google.com>

### CODE REVIEW

**Comment by kyessenov** [comment]
Dead link in the chinese part of the website. Need help getting around it.

**Comment by kyessenov** [comment]
Ping @geeknoid

**Comment by frankbu** [comment]
Every PR seems to be blocked by this broken stephanieleary.com link. Looks like maybe the article has been removed. 

@geeknoid Should we remove the link or maybe change it to another article on the subject of bad link titles?

@kyessenov I can force merge this PR if you'd like.

File: content/docs/reference/config/policy-and-telemetry/attribute-vocabulary/index.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ deployments will have agents (Envoy or Mixer adapters) that produce these attrib
 | `destination.service.uid`       | string | Unique identifier of the destination service. | `istio://istio-system/services/istio-telemetry` |
 | `destination.service.name`      | string | Destination service name. | `istio-telemetry` |
 | `destination.service.namespace` | string | Destination service namespace. | `istio-system` |
-| `request.headers` | map[string, string] | HTTP request headers. For gRPC, its metadata will be here. | |
+| `request.headers` | map[string, string] | HTTP request headers with lowercased keys. For gRPC, its metadata will be here. | |
```
Comment (frankbu) [inline content/docs/reference/config/policy-and-telemetry/attribute-vocabulary/index.md]: need to fix spelling error
s/lowercased/lowercase/ or add it to .spelling

File: content/docs/reference/config/policy-and-telemetry/attribute-vocabulary/index.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ deployments will have agents (Envoy or Mixer adapters) that produce these attrib
 | `request.total_size` | int64 | Total size of HTTP request in bytes, including request headers, body and trailers. | |
 | `request.time` | timestamp | The timestamp when the destination receives the request. This should be equivalent to Firebase "now". | |
 | `request.useragent` | string | The HTTP User-Agent header. | |
-| `response.headers` | map[string, string] | HTTP response headers. | |
+| `response.headers` | map[string, string] | HTTP response headers with lowercased keys. | |
```
Comment (frankbu) [inline content/docs/reference/config/policy-and-telemetry/attribute-vocabulary/index.md]: same as above

File: content/docs/reference/config/policy-and-telemetry/attribute-vocabulary/index.md (hunk 1 above)
Comment (kyessenov) [reply content/docs/reference/config/policy-and-telemetry/attribute-vocabulary/index.md]: Done

File: content/docs/reference/config/policy-and-telemetry/attribute-vocabulary/index.md (hunk 2 above)
Comment (kyessenov) [reply content/docs/reference/config/policy-and-telemetry/attribute-vocabulary/index.md]: Done

**Pull request merged**